webdriver-spec.html

```diff
@@ -3919,7 +3919,10 @@
  allow lookup of individual elements and collections of elements.
  Element retrieval searches are performed
  using pre-order traversal of the document’s nodes
- that match the provided selector’s expression.
+ that match the provided selector’s expression, and ignore the value
+ of the <a href="#dfn-page-load-strategy" class="internalDFN"
```

Comment:
See #438. If someone found this unclear, and it's not obfuscating things to clarify, it seems wise to help clear things up.